src/raw.rs

```diff
@@ -547,7 +547,7 @@ pub fn save_unsigned(rdb: *mut RedisModuleIO, val: u64) {
 }
 
 #[allow(clippy::not_unsafe_ptr_arg_deref)]
-pub fn string_compare(a: *const RedisModuleString, b: *const RedisModuleString) -> Ordering {
+pub fn string_compare(a: *mut RedisModuleString, b: *mut RedisModuleString) -> Ordering {
```

Comment:
Isn't it strange a comparison function requires `mut` args? It seems to be dictated by the header file,
Also as in redis 7.0
https://github.com/redis/redis/blob/b73de0d9c244cae350b65a08dd7f75c2ae1fb54d/src/redismodule.h#L1048 (args are not `const`)

But it isn't so on redis unstable
https://github.com/redis/redis/blob/af0a4fe20771603f0eab75a1f60748d124cf33c3/src/redismodule.h#L1112
(args are `const`)

Reply:
right but we can't use the unstable yet